PyInstaller/hooks/hook-gevent.py

```diff
@@ -0,0 +1,4 @@
+from PyInstaller.utils.hooks import collect_submodules
+
+# required for gevent.monkey.patch_all in gevent 1.1
+hiddenimports = (collect_submodules('gevent'))
```

Comment:
So this hook shouls be `hook-gevent.monkey`, since if - for any reason `gevent.monkey` is not imported, it is not necessary, right?